src/stacked_borrows.rs

```diff
@@ -407,12 +413,12 @@ impl<'tcx> Stacks {
     ) -> EvalResult<'tcx> {
         assert_eq!(new_bor.is_unique(), new_kind == RefKind::Unique);
         trace!("reborrow for tag {:?} to {:?} as {:?}: {:?}, size {}",
-            ptr.tag, new_bor, new_kind, ptr, size.bytes());
+               ptr.tag, new_bor, new_kind, ptr, size.bytes());
```

Comment:
What is the style for too-long-logging-macro-calls? Manually aligning with the string in the previous line is awkward.

I think usually I prefer
```rust
trace!("format {} string {} with {} stuff",
  arg1, arg2,
  very_long_arg3,
);
```
which is some kind of hybrid between proper one-arg-per-line-block-style and whatever else, to save some vertical space. This is debug code after all, if it takes too much vertical space it makes reading code more annoying.

I'd also be happy with
```rust
trace!(
  "format {} string {} with {} stuff",
  arg1, arg2,
  very_long_arg3,
);
```
which is proper block style *except* for having more than one argument per line.

What do y'all think?

Reply:
Block comments is a given, you're right... as for multiple arguments on the same line, I'm happy with that personally, but it seems the style guide recommends one per line still.

Reply:
The style guide is a guide, not a law, for a reason. ;)

Reply:
Depends on your view... If you run rustfmt on the codebase, it sort of becomes law. ;-) Of course, we don't right now.

Reply:
I recall during the fmt-rfcs process we discussed having an exception for `format!`-like macros to format like this:

```rust
trace!(
    "format {} string {} with {} stuff",
    arg1, arg2, very_long_arg3,
);
```

(Fitting as many args per line after the format string, I think?)

 I don't recall what the official outcome was but I think it's a reasonable convention to adopt for now.

Reply:
Okay, sounds fair.